packages/flutter/lib/src/gestures/multidrag.dart

```diff
@@ -26,6 +26,10 @@ export 'gesture_settings.dart' show DeviceGestureSettings;
 /// Signature for when [MultiDragGestureRecognizer] recognizes the start of a drag gesture.
 typedef GestureMultiDragStartCallback = Drag? Function(Offset position);
 
+/// The default value for inputPointerBehavior.
+/// Accept the input if, and only if, [kPrimaryButton] is pressed.
+bool _defaultButtonAcceptBehavior(int buttons) => buttons == kPrimaryButton;
```

Comment:
I think that this can be a static variable.

Reply:
Can't:
https://dart-lang.github.io/linter/lints/prefer_function_declarations_over_variables.html

Reply:
Sorry, that's my bad... it wasn't a variable, to begin with. Regardless, this function declaration can still be static if I am not mistaken.

Reply:
Ah I agree, I think it could probably be a static method within the drag recognizer class

Reply:
Done :)